app/styles/partials/form.scss

```diff
@@ -18,3 +18,49 @@ label.required {
 input.preview {
   text-overflow: ellipsis !important;
 }
+
+.ui.aligned.segment.basic.forgot-password {
+  margin: 0;
+  padding: 0;
+}
+
+.ui.fluid.large.teal.submit.button.reset-password {
```

Comment:
And please don't use sooo many class names like this `ui.fluid.large.teal.submit.button.reset-password`. How is this readable ?